commit 89c6a4c3c56d753afe348d4035ae3016a0317737
Author: etherealNest <etherealnested@gmail.com>
Date:   Sun May 17 15:12:10 2026 +0200

    Load repository2

diff --git a/fs/namespace.c b/fs/namespace.c
index bf3970d09..2f474b67b 100644
--- a/fs/namespace.c
+++ b/fs/namespace.c
@@ -32,11 +32,22 @@
 #include <linux/fs_context.h>
 #include <linux/shmem_fs.h>
 #include <linux/mnt_idmapping.h>
+#ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
+#include <linux/susfs_def.h>
+#endif // #ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
 
 #include "pnode.h"
 #include "internal.h"
 #include <trace/hooks/blk.h>
 
+#ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
+extern bool susfs_is_current_ksu_domain(void);
+extern struct static_key_true susfs_is_sdcard_android_data_not_decrypted;
+
+#define CL_COPY_MNT_NS BIT(25) /* used by copy_mnt_ns() */
+
+#endif // #ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
+
 /* Maximum number of mounts in a mount namespace */
 static unsigned int sysctl_mount_max __read_mostly = 100000;
 
